pxr/usd/usdShade/plugInfo.json

```diff
@@ -133,7 +133,10 @@
                         "schemaTypes": [
                             "UsdShadeShader"
                         ]
-                    }, 
+                    },
+                    "MaterialBindingApiChecker": {
```

Comment:
Checker or not to Checker?
MaterialBindingAPIApplicationValidator, MaterialBindingAPIApplicationCompliance?